Models/Comment.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.ComponentModel.DataAnnotations.Schema;
+using Microsoft.EntityFrameworkCore;
+
+namespace Blog.Models
+{
+
+    public class Comment
```

Comment:
like every other model, this should extend BaseEntity, similar to my comment on Blog class,